06-building-a-program-start-to-finish/writing-tests/output/tests.js

```diff
@@ -0,0 +1,89 @@
+const truebook = require('../example.json');
```

Comment:
This file (tests.js) should be named according to the convention for naming test files